src/domain/event/EventSummaryPage.tsx

```diff
@@ -146,6 +147,9 @@ const EventSummaryPage: React.FC = () => {
     setShowPublishModal(true);
   };
 
+  //temporary accessibility change pt-598
+  const isAdvancedPublish = false;
```

Comment:
Nice way to disable this. Would it be ever nicee to have this outisde of the component as a "constant" like we have this: `const PAST_OCCURRENCE_AMOUNT = 4;`. 

`const IS_ADVANCED_PUBLISH = false;`

Reply:
This might be also a good place to move that publish section code to its own component (I think it could be in the same file). Just for readability so it is easier to visually see the part that is actually for publishing

Reply:
good comments